### ISSUE TITLE
Custom screen aspect ratio Y axis offset doesn't work in portrait mode

### ISSUE THREAD

**Issue by HFPasfho** [libretro/RetroArch#16733]
## Description

To change screen position you have to use custom screen aspect ratio. Screen offsets works in landscape mode but in portrait mode Y axis offset is ignored as if it's always set to 0. In the meantime horizontal offset works in both modes.

### Expected behavior
For the time being Y axis offset should work in portrait mode.
In the future, changing the screen position and on-screen buttons layout should be possible from within the app, not using tricks with custom aspect ratios and user-made button layout online generator.

### Actual behavior

Y axis offset work only in landscape mode.

### Steps to reproduce the bug

1. Go to settings
2. Select video
3. Select scaling
4. Change aspect ratio to custom
5. Change Y axis offset
6. No offset in portrait mode

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.19.1/965ffcf1be

### Environment information

- OS: iPadOS 17.5.1

**Comment by warmenhoven** [comment]
#16716, #16305

**Comment by frootloop88** [comment]
fixed?

**Comment by frootloop88** [comment]
what are the tricks go et it to work?

**Comment by frootloop88** [comment]
> I believe this is fixed in the nightly builds but won't be fixed in the stable App Store builds until we push a new update.

Are they ipa so I can siseload them with altstore? Can’t find them

**Comment by warmenhoven** [comment]
Yes, this is fixed in the nightly build and will be in the next release.

**Comment by frootloop88** [comment]
> Yes, this is fixed in the nightly build and will be in the next release.

yeah its fixed, thanks. Top shelf for Apple TV isn't working though but I guess it's because it's not an App Store release. Glide64 plugin ist still bugged (have to restart the app or I have heavy rendering issues) is this not fixable and should I just use a worse core?

**Comment by warmenhoven** [comment]
> Top shelf for Apple TV isn't working though but I guess it's because it's not an App Store release

That's right. It uses a different bundle id so doesn't have the iCloud container set up for it. (This also makes it look to the OS like a completely different app; you can have the App Store and side load versions installed at the same time.)

> Glide64 plugin ist still bugged

The parallel-rdp plugin is better than gliden64.

**Comment by frootloop88** [comment]
> > Top shelf for Apple TV isn't working though but I guess it's because it's not an App Store release
> 
> That's right. It uses a different bundle id so doesn't have the iCloud container set up for it. (This also makes it look to the OS like a completely different app; you can have the App Store and side load versions installed at the same time.)
> 
> > Glide64 plugin ist still bugged
> 
> The parallel-rdp plugin is better than gliden64.

It does t have adjusted widescreen and no upscail though.

on a side note will you add iCloud support?

**Comment by warmenhoven** [comment]
> iCloud support

For cloud sync you mean? That's already in, yes.

> swo n64 rumble support

No, this is dependent on the mfi driver, I've not seen that there's really anything an application can do for this.

**Comment by frootloop88** [comment]
> > iCloud support
> 
> For cloud sync you mean? That's already in, yes.
> 
> > swo n64 rumble support
> 
> No, this is dependent on the mfi driver, I've not seen that there's really anything an application can do for this.

Where? All I find is webdav

**Comment by warmenhoven** [comment]
Because it requires an iCloud container, it can only be in the App Store build.

**Comment by warmenhoven** [comment]
Incidentally, these conversations might go faster if you find me on the discord, https://discord.gg/C4amCeV

**Comment by dazzzzleart** [comment]
You can set the video driver to opengl, I also encountered such a problem, I hope it can help you.

**Comment by dazzzzleart** [comment]
Setting the video driver to GL solves this problem!! Hope it helps

**Comment by devnoot** [comment]
Setting the video driver to GL doesn't work on IOS devices (or at least doesn't work on my iphone 12)

**Comment by warmenhoven** [comment]
gl works for me on my five different test devices, including ones newer and older than the iPhone 12.

This issue has gotten a bit polluted with things not about the original issue, which is fixed, and in the recent App Store update. This issue should be closed. Any other problems that still exist should be filed as new issues.

**Comment by devnoot** [comment]
Stopping in to say this now works in the latest version of RetroArch on IOS
